staging/src/k8s.io/component-base/logs/sanitization/sanitization_test.go

```diff
@@ -0,0 +1,181 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sanitization
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type withDatapolTag struct {
+	Key string `json:"key" datapolicy:"password"`
+}
+
+func datapolItem() interface{} {
+	return withDatapolTag{Key: "hunter2"}
+}
+
+func TestFilter(t *testing.T) {
+	filter := &SanitizingFilter{}
+	testcases := []struct {
+		input  []interface{}
+		output []interface{}
+	}{{
+		input:  []interface{}{},
+		output: []interface{}{},
+	}, {
+		input: []interface{}{
+			"nothing special", "really",
+		},
+		output: []interface{}{
+			"nothing special", "really",
+		},
+	}, {
+		input: []interface{}{
+			datapolItem(),
+		},
+		output: []interface{}{
+			"Log message has been redacted. Log argument #0 contains: [password]",
+		},
+	}, {
+		input: []interface{}{
+			"nothing special", datapolItem(),
+		},
+		output: []interface{}{
+			"Log message has been redacted. Log argument #1 contains: [password]",
+		},
+	}}
+	for _, tc := range testcases {
+		output := filter.Filter(tc.input)
+		if !assert.ElementsMatch(t, tc.output, output) {
+			t.Errorf("Unexpected filter output for %v, want: %v, got %v", tc.input, tc.output, output)
+		}
+	}
+}
+
+func TestFilterF(t *testing.T) {
+	filter := &SanitizingFilter{}
+	testcases := []struct {
+		inputFmt  string
+		input     []interface{}
+		output    []interface{}
+		outputFmt string
```

Comment:
At least for me reverted order of outputs makes reading those test cases a bit harder.

Reply:
agreed, swapped them around, (and also the FilterS testcases)